WcaOnRails/app/models/competition.rb

```diff
@@ -640,7 +640,11 @@ def in_progress?
   end
 
   def is_over?
-    !end_date.nil? && end_date < Date.today
+    # We treat "today == last day of competition" as the competition is over.
+    # Once we know the schedules and timezones of competitions, we'll be able
+    # to do something more accurate here.
+    # See https://github.com/thewca/worldcubeassociation.org/issues/737
+    !end_date.nil? && end_date <= Date.today
```

Comment:
> Why police this small thing, especially since it makes their job harder and complicates out code?

Good point. Let's just nuke this validation.

I do rather like `is_probably_over?`, as it will trigger the necessary additional thinking that everyone using this method should go through. I will make that change.